js/piwik.js

```diff
@@ -7208,14 +7215,14 @@ if (typeof window.Piwik !== 'object') {
              */
             this.requireConsent = function () {
                 if (!this.hasRememberedConsent()) {
-                    configRequireConsent = true;
+                    configHasConsent = false;
```

Comment:
here I wonder if we should set `configHasConsent = this.hasRememberedConsent();`?

Just in case for some reason it was set to false before, then it would be set to `true` afterwards.

Reply:
Couldn't hurt. Assuming it's always going to be called before the other consent functions.